pyproject.toml

```diff
@@ -37,43 +37,30 @@ requests = "^2.31.0"
 uWSGI = "^2.0.23"
 
 [tool.poetry.group.dev.dependencies]
-black = "24.4.2"
 django-extensions = "3.2.3"
 factory-boy = "3.3.0"
-flake8 = "7.0.0"
-flake8-blind-except = "0.2.1"
-flake8-debugger = "4.1.2"
-flake8-docstrings = "1.7.0"
-flake8-isort = "6.1.1"
-flake8-string-format = "0.3.0"
-flake8-tuple = "0.4.1"
 gitlint = "0.19.1"
-ipython = "8.12.3"
-isort = "5.13.2"
+ipython = "8.26.0"
 pdbpp = "0.10.3"
-pre-commit = "3.5.0"
+pre-commit = "3.7.1"
 pytest = "8.2.2"
-pytest-cov = "4.1.0"
+pytest-cov = "5.0.0"
 pytest-django = "4.8.0"
 pytest-env = "1.1.3"
-pytest-factoryboy = "2.6.0"
-pytest-mock = "3.12.0"
+pytest-factoryboy = "2.7.0"
+pytest-mock = "3.14.0"
 pytest-randomly = "3.15.0"
 python-semantic-release = "7.33.3"
-requests-mock = "1.11.0"
+requests-mock = "1.12.1"
 snapshottest = "0.6.0"
+ruff = "^0.5.1"
 
-[tool.isort]
-skip = [
-  "migrations",
-  "snapshots",
-]
-known_first_party = "ebau_gwr"
-multi_line_output = 3
-include_trailing_comma = true
-force_grid_wrap = 0
-combine_as_imports = true
-line_length = 88
+[tool.ruff.lint]
+extend-select = ["I"]
```

Comment:
this enforces isort-like behavior. We don't use this in inosca/ebau yet, but I think it makes sense?